Style.css

```diff
@@ -0,0 +1,74 @@
+body {
+  background-image: url(Header-pic.png);
+  background-repeat: repeat;
+}
+
+header {
+  background-color: aliceblue;
+  height: 40px;
+  display: flex;
+}
+
+.navbar {
+  font-size: large;
+}
+
+nav a {
+  margin: 10px;
+}
+
+.logo {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-size: larger;
+}
+
+.union {
+  width: 40px;
+  height: 40px;
+}
+
+li {
+  display: inline;
+  margin-left: 10px;
+}
+
+.title {
+  padding-top: 100px;
+  padding-left: 40px;
+  font-size: large;
+  text-align: center;
+}
+
+.title .name {
+  color: rgb(23, 11, 69);
+  font-size: 40px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+}
+
+.job {
+  padding: 10px;
+  color: rgb(27, 168, 19);
+  font-size: 40px;
+  font-family: 'Inter', sans-serif;
+}
+
+.content {
+  display: flex;
+  padding: 40px;
+  font-size: 20px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
```

Comment:
-  [ ] As the previous reviewer suggested, kindly adjust the design of the headline section to match the Figma template for the margin, title color, and background position please fix them.

| Your code  | Figma |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/74744787/212096209-452f07a8-a050-42c6-89bb-3ea6fa5aafdb.png)  | ![image](https://user-images.githubusercontent.com/74744787/212096525-fbb441bc-af66-40bc-90e8-23e4caf03def.png)  |

- [ Optional ] Please fix the margin between the headline title and the toolbar 
![image](https://user-images.githubusercontent.com/74744787/212097023-048b6815-0527-4d15-b530-b354971d0c1d.png)
 
- [ Optional ] the toolbar section doesn't match the design the logo should be aligned to the left while the hamburger button should be aligned to the right also the background of the toolbar section should match the design.

| Your code  | Figma |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/74744787/212098851-0d2e7b44-1329-4ac7-813a-173575e141e8.png)  | ![image](https://user-images.githubusercontent.com/74744787/212098953-5ef64175-a0f5-4fc1-af44-6d676b663679.png)  |